roles/openshift_master/meta/main.yml

```diff
@@ -13,8 +13,6 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_clock
-- role: openshift_docker
-- role: openshift_cli
 - role: openshift_master_certificates
```

Comment:
openshift_cli is still pulled in by openshift_master_certificates, correct? If so, I see no issues here, especially since openshift_cli should be pulling in openshift_docker conditionally.